pkg/config/config_template.yaml

```diff
@@ -873,6 +873,41 @@ api_key:
   #
   # log_file: /var/log/datadog/system-probe.log
 
+{{- if .SecurityModule }}
+
+##########################################
+## Security Agent Runtime Configuration ##
+##########################################
+
+# runtime_security_config:
+  ## @param enabled - boolean - optional - default: false
+  ## Set to true to enable the Security Runtime Module.
+  #
+  # enabled: false
+
+  ## @param socket - string - optional - default: /opt/datadog-agent/run/runtime-security.sock
+  ## The full path to the location of the unix socket where security runtime module is accessed.
+  #
+  # socket: /opt/datadog-agent/run/runtime-security.sock
+
+  ## @param policies - custom object - optional
+  ## Policy names and their policy rules.
+  #
+  # policies:
+    # - name: default-policies
+    #   files:
+    #     - /etc/datadog-agent/security_agent.d/runtime-policies.yaml
```

Comment:
How do we envision configuring this through ENV vars? Should it just be a directory mount with all policies automatically loaded? What is the benefit of having individual files listed here?

Reply:
As suggested we are going to load all the policy files of a path. We will update this PR

Reply:
Great, thank you. If you want to keep it consistent with existing code in compliance, then maybe `runtime_security_config.dir` or `runtime_security_config.policies.dir` or similar?

Reply:
Changed to `runtime_security_config.policies.dir`